lib/src/cpp/utils/XMLManager.cpp

```diff
@@ -236,6 +236,14 @@ void XMLManager::create_node(
 
     // Append new node to parent node
     parent_node->appendChild(last_node);
+
+    // Add text node to correctly set closing tag indentation
+    std::string dummy("\n");
```

Comment:
Perhaps this variable could be renamed into something like `fix_node_identation`

Reply:
In fact the string was already defined in `XMLManager` class: `LINE_BREAK`.